src/lib/components/Tooltip.svelte

```diff
@@ -1,9 +1,14 @@
+<script lang="ts" context="module">
+  let nextTooltipIdSuffix = 0;
```

Comment:
One question, if a tooltip is mounted and then unmounted, does it use a new `nextTooltipIdSuffix`?

Reply:
If it's a new instance of the component, it will get a new id.
I don't think the same instance can be mounted multiple times (can it?) but if it could, it would have the same id.

Reply:
So this number will go up for every new instance. If a page has 10 tooltips it go up to 10, the user navigates away, tooltips are dismounted and when the user comes back, 10 more are rendered with ids 11 to 20, right?

Reply:
Yes, well, only for those tooltips that don't specify in `id`.
Does having the number constantly increase seem concerning to you?

Reply:
Not at all. I wanted to understand the functionality only.

Reply:
OK, thanks!